packages/core/src/Cardano/types/CardanoNode.ts

```diff
@@ -0,0 +1,44 @@
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore
+import { CardanoNodeError, CardanoNodeNotInitializedError } from '../types';
+import { ConnectionConfig } from '@cardano-ogmios/client';
+
+export interface EraSummary {
+  parameters: {
+    epochLength: number;
+    slotLength: number;
+  };
+  start: {
+    slot: number;
+    time: Date;
+  };
+}
+
+export interface CardanoNode {
+  /**
+   * Initialize CardanoNode instance
+   *
+   * @param {ConnectionConfig} connectionConfig Ogmios connection configuration
+   */
+  initialize: (connectionConfig: ConnectionConfig) => Promise<void>;
```

Comment:
It would be better to not couple this interface with `@cardano-ogmios/client` - we have `packages/ogmios` for ogmios-specific implementations. Concrete implementation can have `connectionConfig: ConnectionConfig` injected via construcutor, and/or have additional method(s)

If we need to have life-cycle methods we could extend `Provider` because this interface is Provider-like and is used in a similar fashion within `cardano-services`.